api/clients/v2/config.go

```diff
@@ -61,6 +61,24 @@ type RelayPayloadRetrieverConfig struct {
 	RelayTimeout time.Duration
 }
 
+// ValidatorPayloadRetrieverConfig contains an embedded PayloadClientConfig, plus all additional configuration values
+// needed by a ValidatorPayloadRetriever
+type ValidatorPayloadRetrieverConfig struct {
+	PayloadClientConfig
+
+	// The timeout duration for calls to retrieve blobs from a given quorum.
+	RetrievalTimeout time.Duration
```

Comment:
is this for each individual call to a single node? Or is this a global timeout for retrieving the entire blob from all nodes?

If for single calls might be more accurate to say "to retrieve chunks"

Reply:
This is the amount of time we will wait while trying to retrieve chunks from a given quorum, and reassemble the blob (so, the first option you said)

I will modify this doc to be clearer

Reply:
[8be41aa7](https://github.com/Layr-Labs/eigenda/pull/1194/commits/8be41aa7b9f639e44e894f4d44669aa441910ccd)